### ISSUE TITLE
Request: Cargo (Rust package manager)

### ISSUE THREAD

**Issue by mondeja** [simple-icons/simple-icons#11083]
### Brand Name

Cargo

### Website

https://crates.io

### Popularity Metric

[12.1k ⭐ on Github](https://github.com/rust-lang/cargo)

### Official Resources for Icon and Color

I've not found the vector, but in the [Rust foundation guidelines page](https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/) they refer to the vector version along with the bitmap one.

Hard to vectorize bitmap there, at https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/

There is an open issue from a long time requesting to create a more "rustic" icon for Cargo at https://github.com/rust-lang/cargo/issues/4446, probably worth to take into account. 

### Additional Comments

Quoting [the trademark](https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/):

> TL;DR: Most non-commercial uses of the Rust/Cargo names and logos are allowed and do not require permission; most commercial uses require permission. In either case, the most important rule is that uses of the trademarks cannot appear official or imply any endorsement by the Rust project.

This icon would be great to add for Rust crates version badges because [crates.io](https://crates.io/) uses the Cargo icon in their top navbar.

**Comment by uncenter** [comment]
Duplicate of https://github.com/simple-icons/simple-icons/issues/1485?

**Comment by mondeja** [comment]
Ops, I haven't seen. Perhaps it could be revisit, maybe I'll put a try but this issue doesn't need to be opened. So closing...
